pom.xml

```diff
@@ -384,12 +394,12 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <raml-module-builder.version>30.0.0</raml-module-builder.version>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
I think it is ok to remove these properties. Because java version is already defined in the `maven-compiler-plugin`.

Reply:
I tried removing these during the upgrade. Without having them set explicitly, the compiler defaults to a very old version of the java api and the build breaks.

Reply:
That's strange, it works fine for me locally.

Reply:
I didn't investigate it any further, but I did re-verify the problem just now. 

I'm using opendjk 11 on Ubuntu 20.04 within Windows' WSL.

Reply:
Removing these two properties works *after* upgrading the maven compiler version from 3.5.1 to 3.8.1.